swagger_server/controllers_local/collectioninstrument.py

```diff
@@ -0,0 +1,300 @@
+##############################################################################
```

Comment:
I see the same
```
exercise = self._get_exercise(exercise_id)
        if not exercise:
            return 204, {'text': 'No such batch'}
```
At worst the json result should be extracted for maintainability. This would also apply for {'text': 'OK'}